### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by halles**

### CODE REVIEW

File: src/Form/ValidateEmailForm.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\dpc_user_management\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+class ValidateEmailForm extends FormBase{
```
Comment (G-Rath) [inline src/Form/ValidateEmailForm.php:9]: 🐘 this file could use some formatting :)

File: src/Form/ValidateEmailForm.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     * Returns a unique string identifying the form.
+     *
+     * The returned ID should be a unique string that can be a valid PHP function
+     * name, since it's used in hook implementation names such as
+     * hook_form_FORM_ID_alter().
+     *
+     * @return string
+     *   The unique string identifying the form.
+     */
+    public function getFormId() {
```
Comment (G-Rath) [inline src/Form/ValidateEmailForm.php:20]:
```suggestion
    public function getFormId(): string {
```

This should be fine ay?

File: src/Form/ValidateEmailForm.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return 'validate_email_form';
+    }
+
+
+    /**
+     * @param array $form
+     * @param FormStateInterface $form_state
+     * @return array
```
Comment (G-Rath) [inline src/Form/ValidateEmailForm.php:28]:
```suggestion
     * @param FormStateInterface $form_state
     * @param $parameter (optional)
     * @return array
```

🐘

**Review by G-Rath — APPROVED** [review]

File: src/Form/ValidateEmailForm.php (hunk 3 above)
Comment (halles) [reply src/Form/ValidateEmailForm.php:30]: Interface actually has only 2 parameters and `$parameter` is not used in the form either.

**Pull request merged**